doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con_custom-tasks-overview.adoc

```diff
@@ -78,6 +78,11 @@ In the BPMN modeler, you can configure the following general properties for a se
 |*Assignments*
 |Defines data input and output for the task.
 
+| *Metadata Attributes*
+a| Defines the custom metadata attribute name and value that you want to use for custom event listeners, such as a listener to implement some action when a metadata attribute is present.
+
+The *Metadata Attributes* enable the new `metaData` extensions to BPMN diagrams and modify the behavior of the overall task.
```

Comment:
Again "modify the behaviour" is unclear to me.

Reply:
Please see the above explanation